helixbass/tsc-rust
# Generate diagnostic messages from JSON file

In this PR:
- mimic part of the `scripts/processDiagnosticMessages.js` in the Typescript version to generate a version of `src/compiler/diagnostic_information_map_generated.rs` that includes all diagnostic messages

To test:
If you `cargo install rust-script`, delete `src/compiler/diagnostic_information_map_generated.rs` and then run (from the repo root) `./scripts/process_diagnostic_messages.rs src/compiler/diagnosticMessages.json` it should re-generate `src/compiler/diagnostic_information_map_generated.rs`
Behavior should still be the same as of #25
